include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -525,12 +511,14 @@ struct equal_opposite : public base_turn_handler
 {
     template
     <
-        typename Point1,
-        typename Point2,
+        typename UniqueSubRange1,
+        typename UniqueSubRange2,
         typename OutputIterator,
         typename IntersectionInfo
     >
-    static inline void apply(Point1 const& pi, Point2 const& qi,
+    static inline void apply(
+                UniqueSubRange1 const& range_p,
+                UniqueSubRange2 const& range_q,
```

Comment:
Here they are passed at the beginning of arguments list which is as it was before.